src/assets/Hindi.json

```
@@ -1896,7 +1903,8 @@
         "numberPerWeek": "Number Per Week",
         "youngChildAgeShouldBeLessThanBenAge": "Youngest child age entered can not be greater than beneficiary age",
         "immunizationService": "Immunization Services",
```

Biome (2.4.12)

[error] The key immunizationService was already declared.

(lint/suspicious/noDuplicateObjectKeys)